vms/platformvm/api/camino.go

```diff
@@ -22,7 +22,13 @@ import (
 	"github.com/ava-labs/avalanchego/vms/types"
 )
 
-var errNonExistingOffer = errors.New("non existing deposit offer")
+var (
+	errNonExistingOffer             = errors.New("non existing deposit offer")
+	errWrongUTXONumber              = errors.New("wrong utxo number")
```

Comment:
I think this and next err text needs more context :) I'd write explicitly that its not matching amount of deposit definitions